style.css

```diff
@@ -52,9 +52,10 @@ body {
 } */
 
 .nav {
-  display: inline-block;
-  display: flex;
+  display: flex; 
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/app.creation/pullRequest?prid=6278206)